packages/react/src/components/Pivot/Pivot.base.tsx

```diff
@@ -244,9 +244,10 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef<
       <div role="toolbar" {...divProps} ref={ref}>
         <FocusZone
           componentRef={focusZoneRef}
-          direction={FocusZoneDirection.horizontal}
-          className={classNames.root}
           role="tablist"
+          direction={FocusZoneDirection.horizontal}
+          {...focusZoneProps}
+          className={css(classNames.root, focusZoneProps.className)}
```

Comment:
One last thing, since `focusZoneProps` can be undefined this should actually be:

```suggestion
          className={css(classNames.root, focusZoneProps?.className)}
```

That should get rid of the build error.